packages/node/package.json

```diff
@@ -47,16 +48,13 @@
     "@types/cookie": "0.3.3",
     "@types/etag": "1.8.0",
     "@types/jest": "29.5.0",
-    "@types/test-listen": "1.1.0",
     "@vercel/ncc": "0.24.0",
```

Comment:
Can we remove `@vercel/ncc` too?

Reply:
We can. I'll do it in a separate PR (https://linear.app/vercel/issue/ZERO-1003/remove-vercelncc-dep-from-all-packages-packagejson-files)